### ISSUE TITLE
flake-isort generating false warnings when running flake8 on Windows

### ISSUE THREAD

**Issue by AlexArcPy** [wemake-services/wemake-python-styleguide#224]
# Bug report

`flake-isort` is generating false warnings when running `flake8` with `flake8-isort` installed on Windows 10 machine.

## What's wrong

Having file with the contents:

```python
# -*- coding: UTF-8 -*-
"""Module docstring."""

print('hello')
```

Running

```bash
C:\Temp\SampleEnvIsortOnly\Scripts
(SampleEnvIsortOnly) λ flake8 "C:\Development\Flake8PluginDev\foo.py"
```

on Windows 10 gives

```
C:\Development\Flake8PluginDev\foo.py:2:1: I001 isort found an import in the wrong position
C:\Development\Flake8PluginDev\foo.py:4:1: I003 isort expected 1 blank line in imports, found 0
```

when there are really no imports in the file.

Running 

```bash
(SamplePyEnv) linuxuser@LinuxMachine:~/Development/SamplePyEnv$ flake8 /home/linuxuser/Development/Sample/foo.py
```

on the file with the same contents on Linux Ubuntu does not produce any warnings.

## How is that should be

The `flake8-isort` should not generate these false warnings for file with the provided content on Windows machines.

### flake8 information

Contents of `flake8 --bug-report`:

```
(SampleEnvIsortOnly) λ flake8 --bug-report
{
  "dependencies": [
    {
      "dependency": "setuptools",
      "version": "39.0.1"
    }
  ],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.6.6",
    "system": "Windows"
  },
  "plugins": [
    {
      "is_local": false,
      "plugin": "flake8_isort",
      "version": "2.3"
    },
    {
      "is_local": false,
      "plugin": "mccabe",
      "version": "0.6.1"
    },
    {
      "is_local": false,
      "plugin": "pycodestyle",
      "version": "2.3.1"
    },
    {
      "is_local": false,
      "plugin": "pyflakes",
      "version": "1.6.0"
    }
  ],
  "version": "3.5.0"
}
```

### pip information

Contents of `pip freeze`:

```
(SampleEnvIsortOnly) λ pip freeze
flake8==3.5.0
flake8-isort==2.5
isort==4.3.4
mccabe==0.6.1
pycodestyle==2.3.1
pyflakes==1.6.0
testfixtures==6.3.0
```

**Comment by AlexArcPy** [comment]
@sobolevn , I think you may be interested in this, too, if you care for Windows users running the `wemake-python-styleguide` linter :)

**Comment by AlexArcPy** [comment]
@sobolevn , thanks for getting back on this, appreciated. I've changed my working directory to the file that contains `foo.py` I run `flake8` on (to avoid having a `setup.cfg` file inside Python venv. I've created a `setup.cfg` in the current directory with the following content:

```
[flake8]
max-complexity = 6
max-line-length = 80
enable-extensions = G

[isort]
multi_line_output = 3
include_trailing_comma = true
default_section = FIRSTPARTY
line_length = 80
```

I am still getting the same warnings

**Comment by sobolevn** [comment]
Can you please make a reproduction repo?

**Comment by AlexArcPy** [comment]
Thanks for giving a hand on this. I'd love to provide a repo, but a bit unsure on what should be included?

* foo.py
* setup.cfg
* requirements.txt
* ?

**Comment by sobolevn** [comment]
Everything that is required to reproduce your error. It may include all these files + a quick readme about what to do.

**Comment by AlexArcPy** [comment]
Done! https://github.com/AlexArcPy/flake8-isort-debug Let me know if you need more details from me!

**Comment by sobolevn** [comment]
@AlexArcPy sorry for the long reply.

It works fine for me:

```~/Desktop/flake8-isort-debug  master ✗                                         
» source .venv/bin/activate

(.venv) ~/Desktop/flake8-isort-debug  master ✗                                 
» which flake8
/Users/sobolev/Desktop/flake8-isort-debug/.venv/bin/flake8

(.venv) ~/Desktop/flake8-isort-debug  master ✗                                 
» flake8 foo.py            
```

**Comment by sobolevn** [comment]
@AlexArcPy can this be closed? Have you figured out what was wrong?

**Comment by AlexArcPy** [comment]
I've tried various things but this seems to be specific to my Windows 10 environment. Since if I understood it right the appveyor tests on Windows pass, this issue could be closed and no further action is required in this regard.
